src/components/ResourceCard/AddResource/ResourceSelectActivity.js

```diff
@@ -69,16 +68,14 @@ let ResourceSelectActivity = (props) => {
             <div
               className="activity-img"
               style={{
-                backgroundImage: `url(${global.config.laravelAPIUrl}${activity.image})`,
+                backgroundImage: `url(${activity.image})`,
```

Comment:
Do we have full url on activity image? Not relative path?

Reply:
full url will work

Reply:
No reply for this?